### PR TITLE
use Fieldset in credential library create page

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

File: ui/admin/app/components/form/credential-library/index.hbs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    {{#if @model.errors.name}}
+      <field.errors as |errors|>
+        {{#each @model.errors.name as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.input>
+
+  <Rose::Form::Fieldset>
```
Comment (DhariniJeeva) [inline ui/admin/app/components/form/credential-library/index.hbs:66]: @gsusmi  L65-L82 are the only changes I made, not sure why git shows all the other files I didn't touch?

**Comment by DhariniJeeva** [comment]
@calcaide @randallmorey @gsusmi  Thoughts on making fieldset title `semi-bold`?  Currently, in the OIDC form except for a few (Signing Algorithms, Allowed Audiences, etc), all the other fields are `semi-bold` and the new credential store creation form also uses semi-bold fields as well.

File: ui/admin/app/components/form/credential-library/index.hbs (hunk 1 above)
Comment (gsusmi) [reply ui/admin/app/components/form/credential-library/index.hbs:66]: Try rebasing onto `credential-library-create` and pick your commit to force push changes back onto this branch. It should cleanup history.

**Comment by gsusmi** [comment]
Since we started moving fieldset to match field label styles, it makes sense to keep both in styles in sync. 👍 to apply `semi-bold`.

**Comment by gsusmi** [comment]
@DhariniJeeva , should https://github.com/hashicorp/boundary-ui/pull/655 be closed without merging as it duplicates this PR?

**Comment by calcaide** [comment]
As mentioned [in a comment by Susmitha](https://github.com/hashicorp/boundary-ui/pull/656/files#r674778515) I think you need to perform a rebase. This PR is showing lots of changes introduced before this, so it is hard to determinate what changes you introduced that need review.

Regarding the fieldset being `semi-bold`: Indeed with you and Susmitha, best to keep styles in sync and get the consistency.

**Pull request closed without merging**